src/engraving/dom/spanner.h

```diff
@@ -220,31 +220,31 @@ class Spanner : public EngravingItem
     void setNoteSpan(Note* startNote, Note* endNote);
 
     EngravingItem* startElement() const { return m_startElement; }
-    EngravingItem* endElement() const { return m_endElement; }
+    virtual EngravingItem* endElement() const { return m_endElement; }
```

Comment:
Is it necessary to override all of these methods? Does it become too annoying if you just ensure that m_endElement is always null for LV ties and let the other methods (which work with m_endElement) deal with it?

Reply:
I thought this was pretty ugly, glad to get rid of it.  It does work when only overriding `setEndNote` and `setEndElement` to which I've added a stronger assertion.